internal/pkg/cli/pipeline_delete.go

```diff
@@ -176,21 +179,34 @@ func (o *deletePipelineOpts) askAppName() error {
 }
 
 func (o *deletePipelineOpts) getSecret() error {
-	path, err := o.ws.PipelineManifestLegacyPath()
-	if err != nil {
-		return fmt.Errorf("get path to pipeline manifest: %w", err)
+	// Look for default secret name for GHv1 access token based on default pipeline name.
+	o.ghAccessTokenSecretName = o.pipelineSecretName()
+	output, err := o.secretsmanager.DescribeSecret(o.ghAccessTokenSecretName)
+	if err == nil {
```

Comment:
what do you think about swapping the error handling to handle `err != nil` first?

Reply:
I had it that way originally, but then I had two instances of logging that long info line and resetting `o.ghAccessTokenSecretName` back to an empty string (once if secret not found, again if secret found but without the matching tags), so this seemed slightly more efficient.

Reply:
I may switch it again if I get rid of the log.info altogether....